code_review.md

```diff
@@ -0,0 +1,98 @@
+* Use the pull request description template [3]. You only need to answer the relevant questions. Pro tip: use the bookmarklet [11] or chrome extension [12] for pull request description templates.
+* Annotate your code reviews. Add comments anywhere that you feel will help the reviewer understand your code.
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+change.")
+* Don't take it personally. The review is about the code, not you.
+* Explain why the code exists. ("It's like that because of these reasons. Would
+it be more clear if I rename this class/file/method/variable?")
+* Seek to understand the reviewer's perspective.
+* Try to respond to every comment.
+* Wait to merge the branch until someone has :+1:'d your PR and Continuous Integration
+tells you the test suite is green in the branch.
+* Merge once you feel confident in the code and its impact on the project.
+* Delete the backing branch.
+
+# Reviewing Code
+* Code review often and for short sessions. The effectiveness of code review drops after around an hour. Set aside time throughout your day to coincide with breaks in order to not disrupt your flow.
+* Review at least part of the code, even if you can't do all of it.
+* Review the right things, let tools to do the rest. Don't focus on reviewing things that can be reviewed by tools (e.g. Rubocop, jsLint, sass lint, etc). If the tools don't provide good feedback, change the config or improve the tools.
+
+Understand why the code is necessary (bug, user experience, refactoring). Then:
+
+* Communicate which ideas you feel strongly about and those you don't.
+* Identify ways to simplify the code while still solving the problem.
+* If discussions turn too philosophical or academic, move the discussion offline. In the meantime, let the
+author make the final decision on alternative implementations.
+* Offer alternative implementations, but assume the author already considered
+them. ("What do you think about using a custom validator here?")
+* Seek to understand the author's perspective.
+* Sign off on the pull request with a :thumbsup: or "Ready to merge" comment... always!
+* Compliment/reinforce good practices.
+
+Some suggestions of things to look for when reviewing code:
+
+* Is all the code easily understood? Is code unnecessarily complex ?
+* Can variable/method names be improved?
+* Are classes/files/methods becoming too large? Should they be split into smaller pieces?
+* Does it conform to the agreed coding conventions?
+* Is there any redundant or duplicate code? Is the code as modular as possible?
+* Could the design of the solution be improved (e.g. single responsibility principle, open/closed principle) ?
+* Is the code testable? (i.e. don't add too many or hide dependencies, unable to initialize objects, test frameworks can use methods, do tests exist and are they comprehensive?)
```

Comment:
This should be more rigorous:

``` 
Are there tests? If not, why not? Is there an issue open to add them later?
```

The parenthetical is also confusing, if it's still needed.